### DISCUSSION TITLE
Reorganize Channel Ordering

### DISCUSSION THREAD

**Discussion by alg0trader** [stoatchat/discussions#204] [Feature Suggestions]
Right now it seems the only way to reorganize is to delete and re-create channels. Please allow the ability to move (i.e. drag) a channel up or down.

![image](https://user-images.githubusercontent.com/691447/139493744-14756026-ede4-4dfb-8723-7945ee43eb23.png)

**Comment by insertish** [comment]
Tracked by https://github.com/revoltchat/revite/issues/24

**Reply by insertish** [reply]
P.S. you can just put them in a category to order on the current production client
